backend/app/models/role.py

```diff
@@ -6,3 +6,4 @@
 class Role(Base):
     role_id = Column(Integer, primary_key=True, index=True)
     role_name = Column(String, nullable=False)
+    linked_skill = Column(Integer, ForeignKey("skill.skill_id")) 
```

Comment:
Same EOF issue.
This role is the access control role for LMS, not the job "role" that we have. This linked skill shouldn't be here

Reply:
We need to create a new table for job role, this "role" refers to Staff/HR/Manager.